release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -2769,7 +2762,9 @@ link:https://access.redhat.com/solutions/6877401[{product-title} 4.10.8 containe
 [id="ocp-4-10-8-removed-feature-azure-mint-mode"]
 ==== Removed features
 
-Starting with {product-title} 4.10.8, support for Google Cloud Platform Workload Identity has been removed from {product-title} 4.10 for the image registry. This change is due to the discovery of link:https://bugzilla.redhat.com/show_bug.cgi?id=2069807[an adverse impact to the image registry]. For more information, see xref:../release_notes/ocp-4-10-release-notes.adoc#ocp-4-10-removed-feature-auth-gcp-workload-identity[Image registry support for Google Cloud Platform Workload Identity removed].
+Starting with {product-title} 4.10.8, support for Google Cloud Platform Workload Identity has been removed from {product-title} 4.10 for the image registry. This change is due to the discovery of link:https://bugzilla.redhat.com/show_bug.cgi?id=2069807[an adverse impact to the image registry].
+
+With {product-title} 4.10.21, support for using GCP Workload Identity with the image registry is restored. For more information about the status of this feature between {product-title} 4.10.8 and 4.10.20, see the related link:https://access.redhat.com/articles/6898641)[Knowledgebase article].
```

Comment:
One typo, one wrong ')' at the end
https://access.redhat.com/articles/6898641)

Reply:
Sorry about that! Thought I changed that. Thanks for catching! Done! :)